src/pages/global.scss

```diff
@@ -0,0 +1,28 @@
+$primary-color: #2b6099;
```

Comment:
O mais comum no caso de arquivos de estilo globais, é que se crie uma pasta dentro de src chamada styles e se coloque lá tudo que for global.